bundle/src/main/java/aem/pokego/lure/web/servlets/AddPokeStopServlet.java

```diff
@@ -34,18 +36,22 @@
     private static final Logger log = LoggerFactory.getLogger(AddPokeStopServlet.class);
     
     @Override
-    protected void doPost(SlingHttpServletRequest request, SlingHttpServletResponse response) 
-                                                throws ServletException, IOException {
-        response.setContentType("application/json");
+    protected void doPost(@Nonnull SlingHttpServletRequest  request,
```

Comment:
It's declared that way in the parent class, and IntelliJ prompts to fix it so the overrides are consistent.